src/code/z_parameter.c

```diff
@@ -226,182 +229,195 @@ void func_80082644(PlayState* play, s16 alpha) {
         }
     } else {
         if (interfaceCtx->aAlpha != 255) {
-            interfaceCtx->aAlpha = alpha;
+            interfaceCtx->aAlpha = risingAlpha;
         }
     }
 }
 
-void func_8008277C(PlayState* play, s16 maxAlpha, s16 alpha) {
+/**
+ * Slowly diminish button alphas on the hud
```

Comment:
```suggestion
 * Lower button alphas on the HUD to the requested value
```

("slowly" is not up to this function)